lib/MetaCPAN/Server/QuerySanitizer.pm

```diff
@@ -50,10 +50,13 @@ sub _scan_hash_tree {
     my $ref = ref($struct);
     if ( $ref eq 'HASH' ) {
         while ( my ( $k, $v ) = each %$struct ) {
-            if ( $k eq $key ) {
-                MetaCPAN::Server::QuerySanitizer::Error->throw(
-                    message => qq[Parameter "$key" not allowed], );
-            }
+            # Mickey: disabling this check for 'script' key since
+            #         for ES 1.7 I need to use it in the
+            #         function_score syntax
```

Comment:
Could you elaborate on why this needs to be done?  Internally we can certainly use `script` but allowing it via the API would expose us to exploits.